### DISCUSSION TITLE
Does SeaORM block/reduce the usage of prepared statements?

### DISCUSSION THREAD

**Discussion by shanesveller** [SeaQL/sea-orm#285] [Q&A]
When working with `sqlx` directly through macros like `query_as!`, my understanding is that this also winds up using prepared statements. Does SeaOrm/SeaQuery reduce or eliminate this benefit, or is it still behaving that way under the hood?

**Comment by tyt2y3** [comment]
Yes, at least by design, for all dynamic queries you are running, if the generated SQL is the same, it'd reuse the same prepared statement, because it's using the same sqlx function under the hood.
